### ISSUE TITLE
Ability to request a Pokebattler raid party

### ISSUE THREAD

**Issue by Jahrern** [PokeNavBot/Issue-Tracker#503]
**Is your feature request related to a problem? Please describe.**
I wouldn't call it a problem, but for people that don't have a server full of pokebattler users, the ability to use a command to request a pokebattler raid party would be better then to just have an on/off option. 

**Describe the solution you'd like**
A command to request a pokebattler raid party. Something along the lines of "$pbr"

**Describe alternatives you've considered**
Can't say I have thought about any other alternatives. 

**Additional context**
Don't think there is much else to say, other then the fact that I think that an option other then just turning the feature on/off would make it a lot more versatile, and would fit any server out there.

**Comment by maconleary** [comment]
I agree with this. Take up in our community isn't high enough to have it on by default, but I'd like to be able to turn it on for the occasional savvy raid group.
